constructor/fcp.py

```diff
@@ -21,6 +21,16 @@
                               download, env_vars, groupby, read_paths_json, all_channel_urls)
 
 
+def getsize(filename):
+    """Return the size of a file, reported by os.lstat as opposed to os.stat."""
+    # Symlinks might be reported as "not found" if they are provided by a
+    # package's dependencies
+    # We use lstat to obtain the size of the symlink, as opposed to the
+    # size of the file it points to
+    # https://github.com/conda/constructor/issues/311
+    return os.lstat(filename).st_size
```

Comment:
Nice. Can we confirm that this works across all supported operating systems?

Reply:
> On platforms that do not support symbolic links, this is an alias for stat().

Reply:
I added the same information to the source. I hope that is enough.

Reply:
Thanks!